src/Microsoft.CodeAnalysis.Razor.Workspaces/AcceptedCharacters.cs

```diff
@@ -3,7 +3,7 @@
 
 using System;
 
-namespace Microsoft.VisualStudio.LanguageServices.Razor
+namespace Microsoft.CodeAnalysis.Razor
```

Comment:
> Your strategy assumes/requires that we don't make any changes to the services as they are currently specified. I'm not sure that's the case.

How does it make that assumption?

If there's one thing that's for-sure in my mind it's that the syntax tree like contracts don't belong in the VisualStudio.LanguageServices.Razor assembly. What those contracts are could definitely change; heck, we could end up having an entirely different cross-plat SyntaxTree introspection API. However, there's no current plan to overhaul what these APIs currently are (at least not concretely) so it'd be nice to at least put them in assemblies that make sense and iterate from there.

The `Microsoft.VisualStudio.LanguageServices.Razor` assembly has become a dumping ground for all things tooling. In order to get things like API multi-targeting, TagHelper resolution, VS for mac and editor loop ownership in the right form. Having the structure in the (currently) right form will allow us to question tooling based decisions to ensure we don't limit ourselves in the long run.